webapp/channels/src/components/common/hooks/useShowAdminLimitReached.ts

```diff
@@ -1,87 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-// import React from 'react';
-import {useIntl} from 'react-intl';
-import {useDispatch, useSelector} from 'react-redux';
-
-import {setNeedsLoggedInLimitReachedCheck} from 'actions/views/admin';
```

Comment:
It seems like this action can be removed along with the corresponding action type and reducer. It's called in the `Login` component, but I can't find anywhere using the Redux state that it sets